payu/models/model.py

```diff
@@ -184,15 +184,11 @@ def expand_executable_path(self, exec):
             return exec
 
         # Check if path set by loading user modules has been defined
-        module_added_path = self.expt.user_modules_path
-        if module_added_path is None:
+        module_added_paths = self.expt.user_modules_paths
+        if module_added_paths is None:
```

Comment:
Should `self.expt.user_modules_paths` just be initialised to an empty list? Or do is it important to be `None` at some stage?

In which case the `module_added_paths` temporary variable wouldn't be required I guess.

Reply:
`None` just indicates that user modules were not loaded (either `expt.setup_modules()` has not been called yet or `$MODULESHOME` is not defined). If `self.expt.user_modules_paths` is used elsewhere for other things, it might be useful for it to stay None.

But it also could simplify the code a little if it was just initialised to `[]`...